tests/test_sources.py

```diff
@@ -80,6 +82,25 @@ def test_yahoo_bars_to_panel_source(self):
             self.assertEqual(next(stocks_iter), event['sid'])
 
 
+class TestSqlSource(TestCase):
+    def test_sql_source(self):
+        source, df = factory.create_test_sql_source()
+        assert isinstance(source.start, pd.lib.Timestamp)
```

Comment:
Ok will do but I took that code from TestDataFrameSource so that class should also be changed.